docs/sphinx/source/user_guide/nomenclature.rst

```diff
@@ -142,6 +142,12 @@ There is a convention on consistent variable names throughout the library:
     solar_zenith
         Zenith angle of the sun in degrees
 
+    spectra, spectra_components
```

Comment:
```suggestion
    spectra
    spectra_components
```

Does listing them on a single line separated by a comma create one term or two terms?  I'd say two terms with the same definition is probably more convenient, and I think that can be achieved by listing them on separate lines.  https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#glossary